cypress/integration/dresses-list.spec.ts

```diff
@@ -1,15 +1,28 @@
-// import {MenuContentPage, DressesListPage} from "../page/index";
+import {MenuContentPage, DressesListPage} from "../page/index";
 
-// describe("the user navigates to the dresses page should", () => {
-//   let menuContentPage: MenuContentPage;
-//   let dressesListPage: DressesListPage;
+describe("the user navigates to the dresses page should", () => {
+  let menuContentPage: MenuContentPage;
+  let dressesListPage: DressesListPage;
 
-//   before(() => {
-//     menuContentPage = new MenuContentPage();
-//     dressesListPage = new DressesListPage();
-//   });
+  before(() => {
+    menuContentPage = new MenuContentPage();
+    dressesListPage = new DressesListPage();
+  });
 
-//   it("show the available dresses", () => {
-//     // ... realiza la prueba
-//   });
-// });
+  it("show the available dresses", () => {
+    // ... realiza la prueba
+
+    // Arrange
+    menuContentPage.visitMenuContentPage();
+    const allDressesNames =
+     ["Printed Dress", "Printed Dress", "Printed Summer Dress", "Printed Summer Dress", "Printed Chiffon Dress"];
```

Comment:
My suggestion is that this const array can be part of the before with the initialization of the pages, as they are not variables but constants